### DISCUSSION TITLE
Looking for maintainers

### DISCUSSION THREAD

**Discussion by Klocman** [Klocman/Bulk-Crap-Uninstaller#289] [General]
Since I'm busy with other things and I no longer use BCUninstaller regularly, I can't regularly update it anymore (severe lack of time). Because of this I decided to look for people willing to maintain this project in my place.

If you are interested in helping out, try making a PR to fix some issue or implement something new. I'll be checking for PRs regularly. If I like what I see, I will add you as a contributor (the more good PRs you submit the better), and optionally include your donate link.

I will still create and vet the releases, at least until I find someone I can fully trust to lead the project in the right direction.

**Comment by ghost** [comment]
so say i wanna help where would i go?

**Reply by Klocman** [reply]
If you are interested in helping out, try making a PR to fix some issue or implement something new.

**Comment by kianc1220** [comment]
Hi, Are u all still looking for maintainers? I'm here to help u all and what should i need to do?
Receive your reply soon.

**Reply by Klocman** [reply]
Hello, you can try fixing some of the open issues first.

**Reply by kianc1220** [reply]
Suree

**Comment by NebulaTechs** [comment]
oh,Now several people have joined？

**Reply by Klocman** [reply]
No one has joined so far.

**Reply by grandpa1946** [reply]
that is sad! i'm not a dev so...

**Comment by jimmycrisp1** [comment]
Im new to coding, developing, etc. However looking to start developing for salesforce, im not sure where to start to help out but if you give me some direction ill be glad to help and learn a bit.

**Reply by Klocman** [reply]
Hello, you can try fixing some of the open issues first.

**Comment by fractalzombie** [comment]
Hello, i'm not new in programming, but i'm backend programmer, what i see i can improve, it's to extend UI and switch it to Blazor and add to it modern design. I try many uninstallers from many companies and it's bad, but this thing is very cool, have cool DB and not bad leftover functionallity. I will create a fork and will check what i can do with it, here is many libraries with components of all UI things and it will add modern WEB 3.0 to APP. It's what i want to do for now, i don't see any functionallity i can add here, maybe only some temp cache cleaner, but it exists in PC Manager from Microsoft.

**Reply by Klocman** [reply]
That'd be nice.

**Comment by anaselfaijah** [comment]
Hey, I'm totally interested! Can definitely help with docs (French and English language) and verification for now. Hoping to code fixes once my skills on C# gets better. By the way, I can code on python. Tell me if you are interested, I'll be glad to help and contribute.

**Reply by Klocman** [reply]
Sounds good, for now you can submit documentation updates through PRs.

**Comment by kraldev0** [comment]
And the end!
Here I come.

**Comment by kraldev0** [comment]
There were errors and language deficiencies, they were corrected.
There is a missing module, I will add it and test it for full functionality.
If I have time tomorrow, I will check again.
In whose repository do you want me to host these?

**Comment by kraldev0** [comment]
[İmage](https://i.hizliresim.com/p53p3o2.jpg)

- Troubleshooting successful.

- Running successful.

- [Now let's update the system and do some infrastructure, but I'm going to sleep now. ] 
- [ Think about it and write your requests. ]

**Reply by Klocman** [reply]
Nice, are you going to start feature PRs to this repo? If that is the case I suggest making each PR for one feature so it's easier to review and modify.

**Comment by ghost** [comment]
Please be careful that the project is not taken over by a malicious maintainer, as happened with the XZ Utils backdoor :(

**Reply by SocialEnginev1** [reply]
Привет, есть софт интересный? для локалок )

**Comment by bc09880** [comment]
Hello, do you know if the project is still moving forward?
I saw that you were looking for maintainers, but I don't know if you found any new ones or what happened.

**Comment by Fireflaker** [comment]
Love this project - and it's even more relevant today. Everything works well for me - but it seems like there is a high dependance of per-case human labor involved. I'm happy to chip in with proofing ratings and classifying detection requests:)

**Comment by lucifer846** [comment]


**Reply by Klocman** [reply]
This has been known for a while now, see https://github.com/Klocman/Bulk-Crap-Uninstaller/discussions/737 

The certificate has been revoked many months ago, so I can't really do anything here, it's just AVs being bad at their job as usual. Funny how the same executable gets flagged now that I don't sign it, but the one with expired signature is apparently still fine.

**Reply by Klocman** [reply]
There is literally nothing I can do about this, the certificate is already invalid and should not be showing the blue UAC dialog. AVs should be detecting the injected dlls in the first place, so send those malicious dlls to the various AV vendor report forms, e.g. https://www.microsoft.com/en-us/wdsi/filesubmission.

**Reply by Twisted-Code** [reply]
That's real nice. I hope Malwarebytes does better than Defender, when it comes to expired signatures, since that's what I use, but it sounds like it doesn't... brilliant.

**Comment by hazeliscoding** [comment]
@Klocman Still looking for maintainers in 2026? I'm willing to comb through some of the issues.

**Reply by Klocman** [reply]
I just checked and sadly I don't have the option to give contributors granular permissions like github orgs do (I wanted to give you triage perms). I'll think about making an org for BCU.

**Reply by Klocman** [reply]
@hazeliscoding I am sorry it took so long but I've sent you an invite.

I've moved BCU to an organization and invited you as a triage collaborator. Please feel free to go through discussions, issues and PRs whenever you have time to spare.

**Reply by hazeliscoding** [reply]
Thank you so much, no worries though. I'll look around more when I have time.

**Comment by Halbour** [comment]
@hazeliscoding king, I'm using the software for so long and I hope someone will maintain it, last commit was 6 months ago 😭

**Reply by hazeliscoding** [reply]
Yeah and also the fact this is the ONLY good uninstaller I've found too makes me low key want to maintain it lol....

**Reply by Halbour** [reply]
Exactly 😭

**Reply by Twisted-Code** [reply]
In a weird way, this may be for the best, since it would be better for the project to stagnate until a worthy successor is found, rather than have a repeat of what happened with XZ Utils 😬

**Reply by Twisted-Code** [reply]
In fact... tbh the lack of recent commits was oddly reassuring when I thought to myself "do I trust this installer given they are somewhat desperate for a maintainer" as it means I just have to trust the current maintainer (though as mentioned above he has stopped signing his code on account of his existing certificate being revoked. Let's hope my Internet connection didn't get MITMed by one of the maintenance guys where I live or somethin)

**Reply by Klocman** [reply]
I'm busy with other things but I still check if things aren't on fire from time to time. Sadly so far I didn't see anyone I could trust with the project (not that there is a need to hand it over in the first place, anyone can create a fork after all).

**Comment by Andy76swe** [comment]
Hope you can find someone to keep this great software going.. a Dark mode would be nice to see in the future ;)

**Comment by Unknown27s** [comment]
May i join here ??? . i really looking forward ya fr

**Comment by One-Simon** [comment]
@Klocman @hazeliscoding 

The Software is awesome and incredibly useful. 
I would love to contribute some time into maintaining & keeping update momentum rolling. 
Currently the repo is in a state where i am not even sure which Prs you would like to be opened, since the currently open ones can´t be checked. 

Would love to not see this software die

**Reply by Klocman** [reply]
I'm fine with most of the PRs. As long as the code makes sense and runs correctly it should be fine to keep.

I've moved BCU to an organization and invited you as a triage collaborator. Please feel free to go through discussions, issues and PRs whenever you have time to spare.

**Reply by Halbour** [reply]
Awesome to see a potential continuation of an awesome FOSS software.

**Reply by One-Simon** [reply]
Great to see it beeing kept alive & thank you for your effort!

**Reply by Andy76swe** [reply]
Hopefully we can get a dark mode now ;)

**Reply by Klocman** [reply]
@One-Simon Can you review the currently open PRs? I'm busy for the foreseeable future.

**Comment by Hew007** [comment]
Can AI now be used to help solve problems, handle maintenance, and even develop new features?

**Reply by hazeliscoding** [reply]
I'd be very hesitant on the develop new features portion, LLMs often think everything is a good idea, and maintenance would need a lot of contexts if the project was not started with AI, would need to transition over to spec-driven development imo for AI to help.

**Reply by NebulaTechs** [reply]
Yes. We must grasp the general direction.

**Reply by Klocman** [reply]
AI cannot maintain a non-toy repository at this point, and probably for a while more now.

**Comment by oofy-01** [comment]
Hi, I live in Bosnia & Herzegovina and I was wondering if you're open for me to translate the BCUninstaller to Bosnian?
Also, I do not know how to code, so if it requires coding then I withdraw my offer

Thanks in advance!

**Reply by Klocman** [reply]
That'd be great. See the contributing file and check out the recently merged PR with Ukrainian translation, you basically have to do the exact same thing.
